.github/workflows/build-and-publish.yml

```diff
@@ -2,7 +2,7 @@ name: Build and Publish
 
 on:
   push:
-    branches: [ 'main', 'master' ]  # Trigger on push to 'main'& 'master' master will be remove step by step
+    branches: [ 'main', 'master' ]
```

Comment:
"master should be removed eventually" might be a useful comment to leave.

Reply:
Maybe. Or maybe it's self-evident.